templates/hpc/index.html

```diff
@@ -157,66 +135,73 @@ <h2>HPC is used across multiple sectors and industries</h2>
           <li class="p-list__item is-ticked">Life sciences</li>
+        </ul>
+      </div>
+      <div class="col-4">
+        <ul class="p-list">
           <li class="p-list__item is-ticked">Manufacturing and industry</li>
           <li class="p-list__item is-ticked">Media and entertainment</li>
           <li class="p-list__item is-ticked">Research and universities</li>
           <li class="p-list__item is-ticked">Weather</li>
         </ul>
-        <p>
-          <a class="p-button--positive" href="/hpc/use-cases">Use cases</a>
-        </p>
       </div>
+      <p>
+        <a class="p-button--positive" href="/hpc/use-cases">See use cases</a>
+      </p>
     </div>
   </section>
-  <section class="p-strip--suru">
-    <div class="row">
-      <div class="col-8">
-        <h2>Solutions for HPC</h2>
-        <p>Embrace the open source mandate in HPC with Canonical.</p>
-        <p>
-          Learn more about how Canonical, a leading provider of commercial support for open source solutions can help you with your challenges as a trusted partner for  high-performance computing and supercomputing.
-        </p>
-        <br />
-        <p>
-          <a class="p-button--positive" href="/hpc/contact-us">Canonical and HPC learn more</a>
-        </p>
-      </div>
-      <div class="col-4 u-hide--medium u-hide--small u-vertically-center">
+
+  <section class="p-strip u-no-padding--bottom">
+    <div class="u-fixed-width u-sv3">
+      <h2>Solutions for HPC</h2>
+      <p>Embrace the open source mandate in HPC with Canonical</p>
+    </div>
+  </section>
+
+  <section class="p-strip u-no-padding--bottom">
+    <div class="u-fixed-width">
+      <div class="u-sv1">
         {{
           image (
-          url="https://assets.ubuntu.com/v1/91b86713-Enterprise_support.svg",
-          alt="",
-          width="288",
-          hi_def=True,
-          loading="auto",
+            url="https://assets.ubuntu.com/v1/3c891a50-Canonical+MAAS.svg",
+            alt="",
+            width="179",
+            height="35",
+            hi_def=True,
+            loading="auto"
           ) | safe
         }}
       </div>
+      <hr>
     </div>
-  </section>
-  <section class="p-strip">
     <div class="row u-equal-height">
       <div class="col-6">
-        <h2 class="p-heading--5">
-          MAAS
-        </h2>
-        <p class="p-heading--2">
-          Cluster deployment and delivery
-        </p>
+        <h2 class="p-heading--3">
+          Cluster management at scale
+        </h3>
+      </div>
+      <div class="col-6">
         <p>
-          The ultimate experience for bare metal resource discovery, management, delivery and observability.
+          <strong>MAAS: super fast cluster deployment and delivery</strong> <br> The ultimate experience for bare metal resource discovery, management, delivery and observability.
         </p>
         <p>
           Deploy HPC and supercomputing clusters with ease.

```

Comment:
Gloria [requested](https://docs.google.com/document/d/1d4E1IA2mEu72q277sihNSkl7QRKsU1ezwA3jVgJ0TCE/edit?disco=AAAAeQXgTmk) this be made a single paragraph